packages/e2e-tests/tests/cypress-04_menu-commands.test.ts

```diff
@@ -59,7 +59,7 @@ test("cypress-04: Test Step buttons and menu item", async ({
 
   await waitForSelectedSource(page, "Link.js");
   await waitFor(async () => {
-    const lineNumber = await getSelectedLineNumber(page, false);
+    const lineNumber = await getSelectedLineNumber(page, true);
```

Comment:
I want to make sure I understand this change correctly.

The lines right above click a jump-to-code button:

https://github.com/replayio/devtools/blob/0c0aff3554f9ad294237906a4e542daa67c858ec/packages/e2e-tests/tests/cypress-04_menu-commands.test.ts#L53-L58

So this test _was_ verifying that line 38 was the _highlighted_ line and now we're verifying that it's the_current_ line (execution point). These aren't necessarily contradictory but they are different and I'm not sure if one is more "correct" than the other in this case.

> The current logic introduced a race condition in which the test would only pass if the expect() ran before the app determined that the highlighted line was in fact on the current point and re-rendered.

I guess it seems like this is what you intended to change so...I guess that's okay?

@markerikson do you have any thoughts here?